doc/source/whatsnew/v0.23.0.txt

```diff
@@ -299,6 +299,28 @@ Supplying a ``CategoricalDtype`` will make the categories in each column consist
     df['A'].dtype
     df['B'].dtype
 
+.. _whatsnew_0230.enhancements.str_cat_align:
+
+``Series.str.cat`` has gained the ``align`` kwarg
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+So far, the method :meth:`Series.str.cat` did not -- in contrast to most of ``pandas`` -- align :class:`Series` on their index before concatenation (see :issue:`18657`).
+The method has now gained a keyword ``align`` which controls this behavior. If ``False``, the behavior will be as previously. If ``True`` and ``others``
```

Comment:
"which controls this behavior" -> "to control alignment"

Reply:
I think the second sentence and the next paragraph can be simplified.

```
The default behavior, not aligning, has not changed. If `align` is not specified, a ``FutureWarning``
is issued and the series are not aligned.

To silence the warning and not align, specify ``align=False``. To silence the warning and align
the Series before concatenating, specify ``align=True``.
```

Reply:
Not sure if I understood correctly; removed part with ``na_rep``.